com.sublimetext.three.yaml

```diff
@@ -17,10 +16,16 @@ finish-args:
   - --socket=wayland
   - --talk-name=org.gnome.SettingsDaemon
   - --filesystem=host
+  - --env=PATH=/app/utils/bin:/app/sublime_merge/bin:/app/bin:/usr/bin
+  - --env=PYTHONPATH=/app/utils/lib/python3.7/site-packages
 
 add-extensions:
+  com.sublimetext.three.SublimeMerge:
+    directory: sublime_merge
+    add-ld-path: lib
+
   com.sublimetext.three.DevUtils:
-    version: "18.08"
+    version: "19.08"
```

Comment:
It seems we'll have to have a corresponding update to https://github.com/flathub/com.sublimetext.three.DevUtils/blob/master/com.sublimetext.three.DevUtils.yaml to build this new version of the extension, would you be interested in making that update?

Alternatively we could leave this out for now and only change it when a new version of the extension is released.

Reply:
Yes, you'll have to update the extension. But maybe it's better to use app-bound versioning for the app extension (e.g. use `stable` or something here)?

Reply:
I'm not entirely sure of the rules for extension versioning but for some reason I thought it has to have the same version string as the runtime.

Reply:
If it's runtime extension, it's version should match the runtime. But here we have app extension, so it probably should match the app (many apps on flathub omit extension version, so it's defaults to the same version as the app).